src/main/java/seedu/fma/model/ModelManager.java

```diff
@@ -36,7 +37,9 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
 
         this.addressBook = new AddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
-        filteredLogs = new FilteredList<>(this.addressBook.getPersonList());
+        // TODO: Sorting order is currently hardcoded
```

Comment:
What do u mean by hardcoded here?

Reply:
its the next line where I sort with a lambda, ideally if we implement sorting in the future we want to properly use SortedList and Comparators instead

Reply:
no need to worry about it now haha